src/platform/imx8/include/platform/clk-map.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2019 NXP
+ *
+ * Author: Daniel Baluta <daniel.baluta@nxp.com>
+ */
+
+#ifndef __INCLUDE_CLOCK_MAP__
+#define __INCLUDE_CLOCK_MAP__
+
+#include <sof/clk.h>
+static const struct freq_table cpu_freq[] = {
+	{666000000, 666000, 0x0}, /* default */
+};
+
+static const struct freq_table ssp_freq[] = {
```

Comment:
If you don't have SSP, why do you keep this?
Probably it's not abstracted enough to build without it, so maybe we should mark it as TODO.

Reply:
@tlauda yes, exactly. Will mark it as TODO.

Reply:
Fwiw, I think this is what @jgunn0262 was working on in a recent PR, i.e. making this generic since SSP is Intel only.